hs/src/Reach/Eval/Core.hs

```diff
@@ -275,6 +280,9 @@ data SLScope = SLScope
   , -- One for the consensus
     sco_cenv :: SLEnv
   , sco_use_strict :: Bool
+  -- Deliberately ignore strict mode when evaluating code
+  -- declared in strict mode
+  , sco_use_unstrict :: Bool
```

Comment:
I feel like there must be a good reason why `unstrict` doesn't just set `use_strict` to `False`?

Reply:
If we were to set `use_strict` to `False` then it could be overwritten by a `'use strict'` statement in the the body of a closure. For example, the below should succeed:

```javascript
const f = () => {
  'use strict';
  const x = 5;
  return true;
}

export const main = Reach.App(() => {
  deploy();
  assert(unstrict(f));
});
```

A similar situation happens for a library that's written in strict mode since the strict mode is captured in the closure environment

Reply:
Ah, yes, that makes sense.